sourcemash/frontend/static/js/routers/app_router.js

```diff
@@ -6,6 +6,7 @@ Sourcemash.Routers.AppRouter = Backbone.Router.extend({
         "categories/:id": "showCategory",
         "saved": "showSaved",
         "browse": "browseFeeds",
+        "categorizer": "showCategorizer",
```

Comment:
But if it's not super obvious, it's not accessible. I just don't see much of a purpose if we're sending a link. That's how they'll get to it. If the fear is the judges click around on the site and then don't know how to get back to the categorizer, leaving a link solely on the splash page isn't going to help if they're playing around with a feed page.

Alex Gerstein
gerstein.alex@gmail.com




On Sun, May 24, 2015 at 7:07 PM -0700, "Scott Gladstone" <notifications@github.com> wrote:












In sourcemash/frontend/static/js/routers/app_router.js:
> @@ -6,6 +6,7 @@ Sourcemash.Routers.AppRouter = Backbone.Router.extend({
>          "categories/:id": "showCategory",
>          "saved": "showSaved",
>          "browse": "browseFeeds",
> +        "categorizer": "showCategorizer",

I don't know the  best way to do it, but it needs to be accessible somehow (even if it's not super obvious).

Maybe it should be a new section on the bottom of the splash page? Materialize's home page would be an example to follow.



—
Reply to this email directly or view it on GitHub.